changelog.d/1114.bugfix

```diff
@@ -0,0 +1,7 @@
+Correct nick validation
```

Comment:
This will end up in the changelog file, so really needs to be on one line. Something along the lines of "Allow nicknames to start with a special character or number according to RFC 2812 § 2.3.1"